myokit/formats/sbml/_importer.py

```diff
@@ -96,53 +102,76 @@ def model(self, path):
         refs = {}
 
         # Handle notes, if given
-        x = dom_child(xmodel, 'notes')
+        x = xmodel.find(self.ns['sbml'] + 'notes')
         if x:
             log.log('Converting <model> notes to ascii')
-            model.meta['desc'] = html2ascii(x.toxml(), width=75)
+            model.meta['desc'] = html2ascii(ET.tostring(x, encoding='unicode'),
+                                            width=75
+                                            )
             # width = 79 - 4 for tab!
 
         # Warn about missing functionality
-        x = dom_child(xmodel, 'listOfCompartments')
+        x = xmodel.find(self.ns['sbml'] + 'listOfCompartments')
         if x:   # pragma: no cover
             log.warn('Compartments are not supported.')
-        x = dom_child(xmodel, 'listOfSpecies')
+        x = xmodel.find(self.ns['sbml'] + 'listOfSpecies')
         if x:   # pragma: no cover
             log.warn('Species are not supported.')
-        x = dom_child(xmodel, 'listOfConstraints')
+        x = xmodel.find(self.ns['sbml'] + 'listOfConstraints')
         if x:   # pragma: no cover
             log.warn('Constraints are not supported.')
-        x = dom_child(xmodel, 'listOfReactions')
+        x = xmodel.find(self.ns['sbml'] + 'listOfReactions')
         if x:   # pragma: no cover
             log.warn('Reactions are not supported.')
-        x = dom_child(xmodel, 'listOfEvents')
+        x = xmodel.find(self.ns['sbml'] + 'listOfEvents')
         if x:   # pragma: no cover
             log.warn('Events are not supported.')
 
         # Ignore custom functions
-        x = dom_child(xmodel, 'listOfFunctionDefinitions')
+        x = xmodel.find(self.ns['sbml'] + 'listOfFunctionDefinitions')
         if x:   # pragma: no cover
             log.warn('Custom math functions are not (yet) implemented.')
 
         # Parse custom units
-        x = dom_child(xmodel, 'listOfUnitDefinitions')
+        x = xmodel.find(self.ns['sbml'] + 'listOfUnitDefinitions')
         if x:
-            self._parse_units(model, comp, x)
+            self._parse_units(model, comp, x, self.ns['sbml'])
+
+        # add time as independent variable (not explicit in SBML format)
+        if bind_time:
+            # add and bind time variable to component
+            comp.add_variable('time')
```

Comment:
What happens if there's already a variable called 'time' ?

Reply:
I was planning on implementing the time read in properly in the next pull request. Apparently it has changed how the SBML versions deal with time, and dependent on the version they assume different base units and different identifiers. 

Should I then remove the time binding for now and incorporate and introduce it with the next pull request?

Reply:
No it's fine to keep it in there, I just think we should have some strategy in case the model contains a variable called "time", because that will cause this line to fail. There's an `add_variable_allow_renaming(name)` method you could use instead, for example?

Reply:
OK. Done.